### ISSUE TITLE
device no longer recognized

### ISSUE THREAD

**Issue by jjm202** [makerdiary/nrf52840-mdk-usb-dongle#86]
Hello,

After going through the steps to program the dongle with a .uf2 image, we can no longer update the firmware with a newer .uf2 image, as the device is no longer recognized (hold the reset button and insert the device into USB slot).  What is the trick so it's recognized as a storage device again so we can drag & drop the new firmware image?

Thanks

**Comment by yloose** [comment]
Did you find a solution?

**Comment by yloose** [comment]
> You can use an external debugger to program it again.

Yeah, thanks that was what I ended up doing. Working well now again. :+1:
